src/rewrite/vct/rewrite/lang/LangSpecificToCol.scala

```diff
@@ -317,17 +336,17 @@ case class LangSpecificToCol[Pre <: Generation](
       case cast: CCast[Pre] => c.cast(cast)
       case sizeof: SizeOf[Pre] => throw LangCToCol.UnsupportedSizeof(sizeof)
 
-      case Perm(a @ AmbiguousLocation(expr), perm)
-          if c.getBaseType(expr.t).isInstanceOf[CTStruct[Pre]] =>
-        c.getBaseType(expr.t) match {
-          case structType: CTStruct[Pre] =>
-            c.unwrapStructPerm(
-              dispatch(a).asInstanceOf[AmbiguousLocation[Post]],
-              perm,
-              structType,
-              e.o,
-            )
-        }
+//      case Perm(a @ AmbiguousLocation(expr), perm)
```

Comment:
Maybe you can just delete this? Otherwise please also comment why it is important to keep